react-flux/js/actions/CartActions.js

```diff
@@ -0,0 +1,18 @@
+var Flux = require('react-flux');
```

Comment:
@lelandrichardson all `.js` and `.jsx` files are missing `'use strict';` line at top, please add it - (https://github.com/voronianski/flux-comparison/blob/master/.jshintrc#L40)